dependencies.py

```diff
@@ -0,0 +1,27 @@
+import os
+import time
+
+Y = set(['yes', 'y', 'YES', 'Y'])
+N = set(['no', 'n', 'NO', 'N'])
+
+os.system("sudo apt-get install figlet")
+time.sleep(1)
+os.system("figlet INSTALLATION")
+agree = input("this tool needs some depedencies so are you sure you want to install those Y / N ")
+
+def agreed():
+  if agree in Y:
+    print("INSTALLING")
+    time.sleep(2)
+    os.system("sudo apt-get update")
+    os.system("sudo apt-get install python3-pip")
+    os.system("pip3 install lolcat")
+    os.system("pip3 install boxes")
+    os.system("pip3 install flask")
+    os.system("pip3 install requests")
```

Comment:
it is just  a simple way to download the dependencies ex: lolcat etc.. 
without typing these commands one by one

Reply:
May be you're missing the point, instead of hardcoding dependencies one by one in the script, they are already mentioned in the requirement.txt file, even if you want to install them from this python script you can simply say `os.system("pip3 install -r requirement.txt")`.

This will avoid duplication of code in case if there is any addition in requirements.

I am not against your script, this script may be modified in case of complex setup requirements. But the code can be simplified for now.

Thanks & regards
Naveen Namani